nixos/modules/services/networking/zerotierone.nix

```diff
@@ -38,6 +49,9 @@ in
     # ZeroTier does not issue DHCP leases, but some strangers might...
     networking.dhcpcd.denyInterfaces = [ "zt0" ];
 
+    # ZeroTier receives UDP transmissions on port 9993 by default
+    networking.firewall.allowedUDPPorts = [ 9993 ];
```

Comment:
We do not open firewall ports by default except for ssh. I am not sure if should make an exception here.

Reply:
The ZeroTier service should have a parameter for port, so `networking.firewall.allowedUDPPorts` could be used at a higher level when really needed.